ntbs-service/wwwroot/css/notificationView.scss

```diff
@@ -58,6 +58,28 @@
   padding-right: 50px;
 }
 
+.overview-treatment-events-table {
+  & th, td {
+    font-size: 16px;
+  }
+  
+  & .date-column {
+    width: 15%;
+  }
+  
+  & .event-column {
+    width: 35%;
+  }
+  
+  & .tb-service-column {
+    width: 35%;
+  }
+  
+  & .case-manager-column {
+    width: 15%;
+  }
```

Comment:
I think these could be made more generic... but that wouldn't be very BEM-y.

I would support either `one-sixth-column` and `two-sixth-column` (I think the rounding is fine), and it wouldn't need to be wrapped in the currently redundant table class.

or if we're BEM-ing

```
episodes-table {
    & episodes-table__date-column {
    }
}
```